### DISCUSSION TITLE
connect evm compatible networks

### DISCUSSION THREAD

**Discussion by noxonsu** [swaponline/MultiCurrencyWallet#4415] [Ideas]
polygon https://coinmarketcap.com/currencies/polygon/ 
wanchain https://medium.com/wanchain-foundation/layer-2-full-evm-compatibility-metamask-wanchains-jupiter-hard-fork-is-nearly-here-d20d20cd4d5b

any else?

**Comment by noxonsu** [comment]
https://coinmarketcap.com/currencies/ethereum-classic/

**Comment by noxonsu** [comment]
https://twitter.com/omgnetworkhq/status/1392113713835040777

**Comment by noxonsu** [comment]
https://chainid.network/chains.json

**Comment by Disti4ct** [comment]
Possibly [tomochain](https://tomochain.com/)
